R/raiss.R

```diff
@@ -223,9 +224,9 @@ invert_mat_eigen <- function(mat, tol = 1e-3){
     
     eigen_mat <- eigen(mat)
     L <- which(cumsum(eigen_mat$values) / sum(eigen_mat$values) > 1-tol)[1]
-    mat_inv <- eigen_mat$vectors[,1:L] %*% 
+    mat_inv <- if(!is.na(L)) eigen_mat$vectors[,1:L] %*% 
```

Comment:
@Travyse if L is NA we should throw and error? we should not return a matrix that is not processed.